### ISSUE TITLE
Build system failure with CMake 3.21

### ISSUE THREAD

**Issue by Blazzes** [skyrim-multiplayer/skymp#62]
When building the project, an error occurs with the installation of vcpkg.

cmake version 3.21.0-rc3
nodeJs version 14.17.3
npm version 6.14.13

![image](https://user-images.githubusercontent.com/49817628/125399320-dc5dd280-e3c9-11eb-8fe6-c6cd2e185682.png)

**Comment by Pospelove** [comment]
Could you please attach `I:\git-installs\skymp\build\vcpkg-manifest-install.log`?

**Comment by Blazzes** [comment]
[vcpkg-manifest-install.log](https://github.com/skyrim-multiplayer/skymp/files/6809988/vcpkg-manifest-install.log)

**Comment by Pospelove** [comment]
Hm, doesn't say anything.

1. Please archive and send the whole `build` directory.
2. Try to store our repo on `C:/` instead of `I:/` and say if it helps.

**Comment by Blazzes** [comment]
I tried to collect on disk C. The result is the same. https://drive.google.com/file/d/1mySFCKbI0LFX6TSehY5d8GUZUrtSV-qF/view?usp=sharing

**Comment by alekcey0211** [comment]
Sorry, did you run the command ' git submodule init` and then `git submodule update`

**Comment by Eli5723** [comment]
There's an issue with cmake 3.21 producing errorless crashes with VCPKG, just downgrade to 3.20.x.

**Comment by ZhenjaMax** [comment]
> There's an issue with cmake 3.21 producing errorless crashes with VCPKG, just downgrade to 3.20.x.

I have the same error and this solution doesn't work for me.

CMake v3.20.5
Node v14.15.4
npm v6.14.10

![image](https://user-images.githubusercontent.com/54914800/125705712-6a2e8d9c-9ad2-48e7-86b8-961d597fd8a5.png)

**Comment by Pospelove** [comment]
This CMake version works for me.
```
C:\Users\Pospelov>cmake --version
cmake version 3.20.0-rc5
```
Maybe updating vcpkg submodule to the latest commit will help. Consider testing that.

**Comment by ZhenjaMax** [comment]
> cmake version 3.20.0-rc5

I've reinstalled CMake and it becomes more weird. CMake doesn't recognize my VS now.

![image](https://user-images.githubusercontent.com/54914800/125708985-4ce1b6f0-0485-4418-ab58-14424270a91c.png)

**Comment by Milord-Freeman** [comment]
I'm not completely sure, but maybe I have a solution.
Visual Studio comes with an Installer where you can select workloads. And it works fine on my computer. I am using VS2019, Cmake 3.20. [] - MSVC_2, and Developer Command Promt for VS 2019, which guarantees working integration of the tools with each other, and correct environment variables. Maybe this will help.

**Comment by ZhenjaMax** [comment]
> I'm not completely sure, but maybe I have a solution.
> Visual Studio comes with an Installer where you can select [workloads](https://docs.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio?view=msvc-160). And it works fine on my PC.

There's some progress after installing language packs but it still show same error.

[skymp/build/vcpkg-manifest-install](https://pastebin.com/dwU1ymC8)

![image](https://user-images.githubusercontent.com/54914800/125806458-959a1cc7-4b1c-4723-9ec5-b8073de1805b.png)

**Comment by Blazzes** [comment]
After many attempts to build the project normally, I finally managed to do it. 
The project was built with this configuration: 
![image](https://user-images.githubusercontent.com/49817628/126020812-8be6bb87-52e8-4aa3-95cd-17c0e2952139.png)
The assembly requires only VS Community Edition and the English language pack installed to it.

**Comment by Pospelove** [comment]
We are going to fix that to work with the latest CMake version, Reopening since the build-system bug still exists.

**Comment by Pospelove** [comment]
Fixed in #85. Feel free to reopen if the problem still exists
